Framework/PythonInterface/plugins/algorithms/BASISReduction.py

```diff
@@ -513,7 +513,7 @@ def _PyExec(self):
                            InputWorkspace=self._samSqwWs)
 
             # additional output
-            if self.getProperty('OutputSusceptibility').value:
+            if self.getProperty('OutputSusceptibility').value is True:
```

Comment:
Is this actually necessary? I thought a bool gets checked against true/false already

Reply:
not really necessary here since `self.getProperty('OutputSusceptibility').value` is guaranteed to be an instance of `Bool`. Here is just a matter of habit